src/doc/reference.md

```diff
@@ -1263,9 +1263,6 @@ be undesired.
 * Exiting without calling destructors
 * Sending signals
 * Accessing/modifying the file system
-* Unsigned integer overflow (well-defined as wrapping)
```

Comment:
Hmm.  You may actually want to leave these two bullet points in: We are continuing to use well-defined *fallbacks* when one's code is compiled without the overflow checking turned on, as discussed in the [Arithmetic Overflow RFC](https://github.com/rust-lang/rfcs/blob/master/text/0560-integer-overflow.md#arithmetic-operations-with-error-conditions); its the part where we say what the "defined result" is for each operation of interest.